ci/jenkins/jobs/macros.yaml

```diff
@@ -231,6 +231,8 @@
           #!/bin/bash
           set -ex
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          # Delete all kind clusters created more than 135 mins ago
```

Comment:
my earlier comment about adding more context to this comment has not been addressed

Reply:
done